en.md

```diff
@@ -0,0 +1,230 @@
+
+The protection of your privacy is extremely important to us. This data protection declaration provides information on how and for what purpose we process personal data and explains the rights to which you are entitled. We are aware of how important personal data is to our users and we uphold all German and European data protection provisions.
+
+## 1. Contact & data protection officer
+In relation to the General Data Protection Regulation (DSGVO), the organisation responsible is: 
+
+Ecosia GmbH  
+Schinkestraße 9  
+12047 Berlin Germany  
+[info@ecosia.org](mailto:info@ecosia.org)
+
+Our Data Protection Officer can be contacted at privacy@ecosia.org or our postal address with the addition of “data protection”.
+
+## 2. Your rights
+You have the following rights with respect to us regarding the personal data relating to you:
+
+ - Right to information
+ - Right to correction or deletion
+ - Right to restriction of processing
+ - Right to refuse processing Right to data transferability
+ - In addition, you have the right to complain to a data protection supervisory authority regarding our processing of your personal data.
+
+## 3. Processing by external service providers
+
+In some cases, we avail of external service providers to process your data. We select and commission these carefully, they are bound by our instructions and inspected on a regular basis.
+
+## 4. Gathering of personal data while visiting our website
+
+In the case of use of the website for purely informational purposes, if you do not – for instance, using our search function – provide us with information in any other way, we gather only the personal data which your browser transmits to our server. If you wish to view our website, we gather the following data, which we technically require in order to show you our website and to guarantee stability and security. The legal basis for the temporary storage of the data is Art. 6 Para. 1 lit. f of the General Data Protection Regulation.
+
+ - IP address
+ - Date and time of enquiry
+ - Difference between time zone and Greenwich Mean Time (GMT)
+ - Content of the request (specific webpage)
+ - Access status / HTTP status code
+ - Respectively transferred data quantity
+ - Website from which the request comes
+ - Browser
+ - Operating system and its surface
+ - Language and version of the browser software.
+
+The data are saved in the logfiles of our system. These data and other personal data of the user are not saved. The data are deleted once they are no longer required to achieve the purpose of your gathering. In the event that the data are saved in log files, this occurs at the most seven days later. 
+
+It is possible to save the data for longer than this. In this case, the users’ IP addresses are deleted or anonymised so that they can no longer be associated with the visiting client.
```

Comment:
Also this is confusing: if we don't store any personal data of the user, we shouldn't be in possess of a data representation of the "visiting client"